client/my-sites/patterns/components/header/index.tsx

```diff
@@ -1,23 +1,180 @@
+import { Substitution, translate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
 import { preventWidows } from 'calypso/lib/formatting';
 import { PatternsSearchField } from 'calypso/my-sites/patterns/components/search-field';
+import type { Category } from 'calypso/my-sites/patterns/types';
 
 import './style.scss';
 
-type PatternsHeaderProps = {
-	description: string;
-	title: string;
+const OPTIONS = {
+	home: {
+		title: {
+			comment: 'HTML title of the page that showcases block patterns',
+			textOnly: true,
+		},
+		header: {
+			comment: 'Header of the page that showcases block patterns',
+			textOnly: true,
+		},
+		description: {
+			comment: 'Description of the page that showcases block patterns',
+			textOnly: true,
+		},
+	},
+	category: {
+		title: {
+			comment: 'HTML title of the page that lists block patterns for a specific category',
+			textOnly: true,
+		},
+		header: {
+			comment: 'Header of the page that lists block patterns for a specific category',
+			textOnly: true,
+		},
+		description: {
+			comment: 'Description of the page that lists block patterns for a specific category',
+			textOnly: true,
+		},
+	},
```

Comment:
The comments will not be picked up in this case, because the string extraction process is based on static analysis of the source code looking up for `translate()` calls and inspecting the parameters and thus the reference to `OPTIONS.category.title`, for example, in will not be evaluated and it's properties will not be picked up.

When introducing new strings or updating existing ones, you can use the artifacts from the Translate job in TeamCity to confirm the references, comments, contexts, etc. are extracted as you'd expect.